GVFS/GVFS.Common/InstallerPreRunChecker.cs

```diff
@@ -177,14 +177,14 @@ protected virtual bool TryRunGVFSWithArgs(string args, out string consoleError)
 
         private bool IsGVFSUpgradeAllowed(out string consoleError)
         {
-            bool isConfirmed = string.Equals(this.CommandToRerun, GVFSConstants.UpgradeVerbMessages.GVFSUpgradeConfirm, StringComparison.OrdinalIgnoreCase);
+            bool isConfirmed = string.Equals(this.CommandToRerun, GVFSPlatform.Instance.Constants.UpgradeConfirmCommandMessage, StringComparison.OrdinalIgnoreCase);
             string adviceText = null;
             if (!this.IsElevated())
             {
-                adviceText = isConfirmed ? $"Run {this.CommandToRerun} again from an elevated command prompt." : $"To install, run {GVFSConstants.UpgradeVerbMessages.GVFSUpgradeConfirm} from an elevated command prompt.";
+                adviceText = GVFSPlatform.Instance.Constants.RunUpdateMessage;
                 consoleError = string.Join(
                     Environment.NewLine,
-                    "The installer needs to be run from an elevated command prompt.",
+                    "The installer needs to be run elevated.",

```

Comment:
Nit: I had trouble parsing this. Perhaps "The installer needs to be run with elevated permissions"?